### PR TITLE
Add spotless autoformatter

### PR DESCRIPTION
**Pull request by jessicasyu**
Add `spotless` to `build.gradle` for autoformatting. To run the formatter:

```bash
./gradlew spotlessApply
```

The linter settings were adjusted to ignore the `WhitespaceAround` changes that the autoformatter changed. The remaining problems are mostly TODOs and missing javadocs.

### CODE REVIEW

**Review by cainja — APPROVED** [review]

File: src/arcade/patch/agent/cell/PatchCellContainer.java [hunk 1]
```diff
@@ -73,41 +80,87 @@ public PatchCellContainer(int id, int parent, int pop, int age, int divisions,
...
         MiniBox parameters = factory.popToParameters.get(pop);
-        
+
         // Make cell.
         switch (parameters.get("CLASS")) {
             default:
             case "tissue":
-                return new PatchCellTissue(id, parent, pop, state, age, divisions, location,
-                        parameters, volume, height, criticalVolume, criticalHeight);
+                return new PatchCellTissue(
```
Comment (cainja) [inline src/arcade/patch/agent/cell/PatchCellContainer.java:109]: I don't love this

File: src/arcade/patch/agent/cell/PatchCellFactory.java [hunk 2]
```diff
@@ -159,80 +153,99 @@ public void createCells(Series series) {
...
-                    population.getDouble("CELL_HEIGHT_STDEV"), random));
-            popToAges.put(pop, new Uniform(population.getDouble("CELL_AGE_MIN"),
-                    population.getDouble("CELL_AGE_MAX"), random));
+
+            popToCriticalVolumes.put(
+                    pop,
+                    new Normal(
+                            population.getDouble("CELL_VOLUME_MEAN"),
+                            population.getDouble("CELL_VOLUME_STDEV"),
+                            random));
```
Comment (cainja) [inline src/arcade/patch/agent/cell/PatchCellFactory.java:220]: this does not spark joy

**Review by cainja — COMMENTED** [review]
Added some comments for some auto-formatting sections that seem a little bit like overkill?

File: src/arcade/patch/agent/cell/PatchCellFactory.java (hunk 2 above)
Comment (jessicasyu) [reply src/arcade/patch/agent/cell/PatchCellFactory.java:220]: What would your preference for formatting this part be?

**Pull request merged**